lib/landing_pages/page.rb

```diff
@@ -12,16 +12,16 @@ def self.required_attrs
     %w(name body).freeze
   end
 
-  def self.discourse_attrs
-    %w(theme_id group_ids category_id).freeze
+  def self.pages_attrs
+    %w(name path parent_id remote email theme_id group_ids category_id inline).freeze
   end
 
-  def self.pages_attrs
-    %w(path parent_id remote menu assets email).freeze
```

Comment:
@cabidop Why is this being changed?

Reply:
As both the `discourse_attrs` and the `pages_attrs` were used exclusively to create the `writable_attrs` field, it looked like a good idea to make them usable in the `zip_exporter.rb` file and avoid hardcoding the attributes there (see my [previous comment](https://github.com/paviliondev/discourse-landing-pages/pull/11#discussion_r1329544057)).

Reply:
As both the `discourse_attrs` and the `pages_attrs` were used exclusively to create the `writable_attrs` field, it looked like a good idea to make them usable in the `zip_exporter.rb` file and avoid hardcoding the attributes there (see my [previous comment](https://github.com/paviliondev/discourse-landing-pages/pull/11#discussion_r1329544057)). In any case, the behaviour for the `writable_attrs` was not changed besides the addition of the new `inline` field.

Reply:
Ok 👍